### DISCUSSION TITLE
Where is Dashboard Manual?

### DISCUSSION THREAD

**Discussion by MahmadSharaf** [mirumee/saleor-dashboard#3092] [General] [answered]
I noticed that the Dashboard manual has been removed from the docs. I confirmed that by finding this ticket https://github.com/saleor/saleor-docs/pull/610. 

So, I wonder what the reason after this decision was, and what is the planned alternative for Dashboard guidance?

**Comment by karolkielecki** [comment, marked as answer]
Hi @MahmadSharaf, yes - we've removed the Dashboard manual. It was outdated in some places. We'll provide an alternative, but currently, we're fully booked with other priorities. Could you please let me know what information you're looking for? Do you have any suggestions on what you would like to see in such guidance?

**Reply by MahmadSharaf** [reply]
Thank you for the info. I would love to support and be a part of this, if this is an option please guide me. 

Other than that, I was looking for managing local apps through the dashboard but found only third party apps. With the documentation guidance, I went with cli option but unfortunately the app created without any permissions. And now I am looking for how to add permissions to this app.

**Reply by peelar** [reply]
Hello @MahmadSharaf 👋 

I recommend taking a look at the ["Apps Quickstart"](https://docs.saleor.io/docs/3.x/developer/extending/apps/quickstart/getting-started) tutorial. It guides you through creating an app, together with setting up its permissions.
